packages/@n8n/db/src/entities/test-case-execution.ee.ts

```diff
@@ -54,4 +54,10 @@ export class TestCaseExecution extends WithStringId {
 
 	@JsonColumn({ nullable: true })
 	metrics: TestCaseRunMetrics;
+
+	@JsonColumn({ nullable: true })
+	inputs: IDataObject;
+
+	@JsonColumn({ nullable: true })
+	outputs: IDataObject;
```

Comment:
1. `IDataObject` is legacy cruft. Is this avoidable? Can we use a better type?
2. Shouldn't these be unionized to `null`?

Reply:
1. I think it represent the type well here, as it can be any type of node parameter value. Or any value the user deems important to record in an object. Do you have another suggestion? 
2. Good point.

Reply:
I guess `GenericValue` could work well here. 
`export type GenericValue = string | object | number | boolean | undefined | null;`
Let me know if you think otherwise